plans/keylime-tests-github-ci.fmf

```diff
@@ -1,5 +1,12 @@
 summary:
   Tests used by Packit/TFT CI on Github
+adjust:
+  - when: distro == centos-stream-9
+    prepare+:
```

Comment:
Is this `+` sign expected here? (also in the next statement)

Reply:
Yes. It means that prepare steps should not be overridden, just extended.

Reply:
Nice, thanks for the info.